tests/spec/modules/options/option-item-boolean.spec.js

```diff
@@ -0,0 +1,46 @@
+
+import { getOptionItemWith } from './index';
+
+describe("spec OptionItem:", () => {
+
+  describe("which has boolean value: ", () => {
+
+    beforeEach(() => {
+      this.optionItem = getOptionItemWith('boolean');
+      document.body.innerHTML = __html__['options.html'];
+    })
+
+    it("throw a error when string is fed into assertValue", done => {
+      try {
+        this.optionItem.assertValue('stststring');
+      } catch (e) {
+        done();
+      }
+    });
+
+    it("nothing when boolean is fed into assertValue", done => {
+      try {
+        this.optionItem.assertValue(false);
+        done();
+      } catch (e) {
+        fail(e);
+      }
+    });
+
+    it("ele.checked expected the value when setValueOnElement is called with a value", () =>{
+      this.optionItem.value = ! this.optionItem.value;
+      this.optionItem.setValueOnElement()
+      expect(this.optionItem.value)
+        .toBe(this.optionItem.getElement().checked);
+    });
+
+    it("return string 'change' when _getListenerType is called", () => {
+      expect(this.optionItem._getListenerType()).toBe('change');
+    });
+
+    it("return boolean value when _getTargetValue is called", () => {
```

Comment:
- change to `return 'true' when _getTargetValue is called with a object which include checked property has true`